charts/govuk-jobs/templates/govuk-mirror-sync-cronjob.yaml

```diff
@@ -106,5 +106,6 @@ spec:
                 - /bin/sh
                 - -c
                 - >-
-                  /s5cmd sync --size-only "/data/${WWW_DOMAIN}" "s3://${BUCKET_NAME}/"
+                  /s5cmd sync --size-only "/data/${WWW_DOMAIN}" "s3://${BUCKET_NAME}/";
```

Comment:
optional: could make this `&&` instead of `;` so that the job fails conspicuously if the first command fails but the second succeeds. (Not saying that's necessarily the right tradeoff — up to you — just mentioning in case you hadn't already considered it.)

Reply:
Ah good catch - I was thinking to this allows it to attempt to sync as much stuff as possible. But I suppose if the first command fails and second succeeds then the job won't bubble up the failure.